app/views/products/show.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class="col-sm-8 col-sm-offset-2">
+  <h1><%= @product.name %> <small><%= number_to_currency(@product.price) %></small></h1>
+
+  <%= image_tag(@product.photo_url, alt: "Image of #{@product.name}", height: 200) %>
+  <%= @product.description %>
+
+  <p>Merchant: <%= @product.user.username %></p>
+  <p>
+    Categories: <% @product.categories.each do |cat| %>
+    <%= cat.name %>
+    <% end %>
+  </p>
+
+  <%= link_to "Add to Cart", new_order_path, { method: :get, id: @product.id, class: "btn btn-default" } %>
```

Comment:
Random note; `link_to` generates an anchor (`<a>`) tag, and, according to the HTTP spec, anchors always send GET requests by default, so the `method: :get` here is likely superfluous.